requirements/base.txt

```diff
@@ -4,5 +4,4 @@
 #
 #    make upgrade
 #
-django==1.11.29           # via -c requirements/constraints.txt, -r requirements/base.in
-pytz==2020.1              # via django
+-e git+https://github.com/edunext/eox-core.git@v2.12.3#egg=eox-core  # via -r requirements/base.in
```

Comment:
We can use eox-core,now, it is in pypi https://pypi.org/project/eox-core/

Reply:
Nice!! Done